app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -53,20 +53,28 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
     final BuildPushMessage buildPushMessage =
         BuildPushMessage.fromJson(json.decode(String.fromCharCodes(base64.decode(data))) as Map<String, dynamic>);
     log.fine(buildPushMessage.userData);
+    log.fine('Updating buildId=${buildPushMessage.build?.id} for builder=${buildPushMessage.build?.result}');
     // Example user data:
     // {
     //   "task_key": "key123",
     // }
+    if (buildPushMessage.userData == null) {
+      log.fine('User data is empty');
+      return Body.empty;
+    }
     final Map<String, dynamic> userData = jsonDecode(buildPushMessage.userData!) as Map<String, dynamic>;
     final String? taskKey = userData['task_key'] as String?;
     if (taskKey == null) {
       throw const BadRequestException('userData does not contain task_key');
     }
+    log.fine('Looking up key...');
```

Comment:
Maybe print taskKey, or some value to track which key its trying to lookup?

Reply:
It's logged in the userData line above